tools/wpt/wpt.py

```diff
@@ -40,13 +40,14 @@ def load_commands():
     return rv
 
 
-def parse_args(argv, commands):
+def parse_args(argv, commands = load_commands()):
     parser = argparse.ArgumentParser()
     parser.add_argument("--venv", action="store", help="Path to an existing virtualenv to use")
     parser.add_argument("--skip-venv-setup", action="store_true",
                         dest="skip_venv_setup",
                         help="Whether to use the virtualenv as-is. Must set --venv as well")
     parser.add_argument("--debug", action="store_true", help="Run the debugger in case of an exception")
+    parser.add_argument("--py3", action="store_true", help="Run with python3. NO command is supported so far.")
```

Comment:
Remove the second sentence. If we start adding metadata about Python 3 support we can make the help explicitly list the supported commands, and print a warning if you try an unsupported one.